modules/merge_waiter/data.tf

```diff
@@ -0,0 +1,14 @@
+data "aws_iam_policy_document" "inline_merge_status_update_policy_doc" {
```

Comment:
Same comment as for file appmesh-sf-task-token.py

Reply:
if it appears in master how come it appears as a change ?